models/src_zendesk.yml

```diff
@@ -331,4 +331,20 @@ sources:
           - name: time_zone
             description: Name of the time zone. 
           - name: standard_offset 
-            description: Standard offset of the timezone (non-daylight savings hours). In `+/-hh:mm` format.
\ No newline at end of file
+            description: Standard offset of the timezone (non-daylight savings hours). In `+/-hh:mm` format.
+      
+      - name: schedule_holiday
```

Comment:
should we add an enabled config? that would be turned off/on with `var('using_schedules')`

Reply:
just so the source table doesn't show up in the DAG if it's disabled

Reply:
Updated to include the config.